docs/faq.rst

```diff
@@ -345,12 +345,13 @@ resolution from that.
 Assuming that all your chunks have a size of :math:`2^{21}` bytes (approximately 2.1 MB)
 and we have a "perfect" hash algorithm, we can think that the probability of collision 
 would be of :math:`p^2/2^{n+1}` then, using SHA-256 (:math:`n=256`) and for example
-we have a 1,000 million chunks (:math:`p=10^9`) (a 1,000 million chunks would be about 2,100TB).
+we have 1e9 chunks (:math:`p=10^9`) (1e9 chunks would be about 2100TB).
```

Comment:
you already have the p=10^9 and e-notation is no good for mathematically less inclined people. 

so just say `1000 million` and everybody knows what it means.